manim/mobject/types/vectorized_mobject.py

```diff
@@ -482,24 +489,24 @@ def add_smooth_curve_to(self, *points):
             ])
         return self
 
-    def has_new_path_started(self):
+    def has_new_path_started(self) -> bool:
         nppcc = self.n_points_per_cubic_curve  # 4
         return len(self.points) % nppcc == 1
 
-    def get_last_point(self):
+    def get_last_point(self) -> Vector:
         return self.points[-1]
 
-    def is_closed(self):
+    def is_closed(self) -> bool:
         return self.consider_points_equals(
             self.points[0], self.points[-1]
         )
 
-    def add_points_as_corners(self, points):
+    def add_points_as_corners(self, points: List[Vector]) -> List[Vector]:
         for point in points:
             self.add_line_to(point)
         return points
 
-    def set_points_as_corners(self, points):
+    def set_points_as_corners(self, points: List[Vector]):
```

Comment:
1. We require indexing so it must be a `Sequence` (but doesn't necessarily have to be a list);
2. returns self so SVGMobject (between double quotes because we can't refer to the class from within itself)
```suggestion
    def set_points_as_corners(self, points: typing.Sequence[Vector]) -> "SVGMobject":
```